source/features/bugs-tab.tsx

```diff
@@ -18,20 +18,18 @@ const getBugLabel = async (): Promise<string | undefined> => cache.get<string>(g
 const isBugLabel = (label: string): boolean => supportedLabels.test(label.replace(/\s/g, ''));
 
 async function countBugsWithUnknownLabel(): Promise<number> {
-	const {repository} = await api.v4(`
-		repository() {
-			labels(query: "bug", first: 10) {
-				nodes {
-					name
-					issues(states: OPEN) {
-						totalCount
-					}
+	const labels = await api.v4repository(`
```

Comment:
First feature first bug found. 😅  I made the same mistake in many places.

Have not reviewed yet.
```suggestion
	const {labels} = await api.v4repository(`
```